src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20161005_refactor_project_users.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="refactor-project-users-1">
+        <sql stripComments="true">
+            alter table BILLING_PROJECT add column OWNER_GROUP varchar(254), add constraint FK_PROJECT_OWNER_GROUP foreign key (OWNER_GROUP) references `GROUP`(NAME)
```

Comment:
this is from before i refactored again. i need to write a brand new liquibase.

it'll be a little trickier because it needs to use an environment variable for group emails